src/probnum/quad/_integration_measures.py

```diff
@@ -0,0 +1,236 @@
+                    constant vector of length ``dim``.
+                2c. Otherwise, if neither of ``domain[0]`` and ``domain[1]`` is a
+                    scalar, error is raised if either of them has length which does not
+                    equal ``dim``.
+        """
+        domain_a_dim = np.size(domain[0])
+        domain_b_dim = np.size(domain[1])
+
+        # Check that given dimensions match and are positive
+        dim_mismatch = False
+        if dim is None:
+            if domain_a_dim == domain_b_dim:
+                dim = domain_a_dim
+            elif domain_a_dim == 1 or domain_b_dim == 1:
+                dim = np.max([domain_a_dim, domain_b_dim])
+            else:
+                dim_mismatch = True
+        else:
+            if (domain_a_dim > 1 or domain_b_dim > 1) and dim != np.max(
+                [domain_a_dim, domain_b_dim]
+            ):
+                dim_mismatch = True
+
+        if dim_mismatch:
+            raise ValueError(
+                "Domain limits must have the same length or at least "
+                "one of them has to be one-dimensional"
+            )
+        if dim < 1:
+            raise ValueError(f"Domain dimension dim = {dim} must be positive.")
+
+        # Use same domain limit in all dimensions if only one limit is given
+        if domain_a_dim == 1:
+            domain_a = np.full((dim,), domain[0])
+        else:
+            domain_a = domain[0]
+        if domain_b_dim == 1:
+            domain_b = np.full((dim,), domain[1])
+        else:
+            domain_b = domain[1]
+
+        # Check that the domain is non-empty
+        if not np.all(domain_a < domain_b):
+            raise ValueError(f"Domain must be non-empty.")
+
+        self.dim = dim
+        self.domain = (domain_a, domain_b)
+
+
+class LebesgueMeasure(IntegrationMeasure):
+    """A Lebesgue measure."""
+
+    def __init__(
+        self,
+        domain: Tuple[Union[np.ndarray, FloatArgType], Union[np.ndarray, FloatArgType]],
+        dim: Optional[IntArgType] = None,
+        normalized: Optional[bool] = False,
+        random_state: Optional[RandomStateArgType] = None,
+    ):
+        super().__init__(
+            dim=dim, domain=domain, random_state=random_state, name="Lebesgue measure"
+        )
+
+        # Set normalization constant
+        self.normalized = normalized
+        if self.normalized:
+            self.normalization_constant = 1 / np.prod(self.domain[1] - self.domain[0])
+        else:
+            self.normalization_constant = 1
+
+        if self.normalization_constant in [0, np.Inf, -np.Inf]:
+            raise ValueError(
+                "Normalization constant is too small or too large. "
+                "Consider setting normalized = False."
+            )
+
+        # Use scipy's uniform random variable since uniform random variables are not
+        # yet implemented in probnum
+        self.random_variable = scipy.stats.uniform(
```

Comment:
You can cast this to a ProbNum random variable if necessary using [`probnum.randvars.asrandvar`](https://probnum.readthedocs.io/en/latest/automod/probnum.randvars.asrandvar.html#probnum.randvars.asrandvar)

Reply:
by just using `asrandvar(scipy.stats.uniform(...))`? What's the benefit?

Reply:
No particular benefit. Reading the comment I thought you were looking for a ProbNum implementation of a uniform random variable.